backend/.env

```
@@ -1,7 +1,7 @@
-# PORT=4000
-# MONGO_URI=mongodb://127.0.0.1:27017/fediverse
-# JWT_SECRET=someSecretKey
-# # BASE_URL=https://7a1db250d5b7.ngrok-free.app
+PORT=4000
+MONGO_URI=mongodb://127.0.0.1:27017/fediverse
+JWT_SECRET=someSecretKey
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the MONGO_URI key

(UnorderedKey)